client/app/telecom/pack/xdsl/access/pack-xdsl-access.html

```diff
@@ -7,424 +7,475 @@ <h3 data-translate="loading"></h3>
-                                <div>
-                                    <span data-ng-bind="access.xdsl.address.numberStreet"></span>
-                                    <span data-ng-bind="access.xdsl.address.street"></span>,
+                            </li>
+
+                            <!-- Access Type -->
+                            <li class="oui-tile__item">
+                                <div class="oui-tile__definition">
+                                    <div class="oui-tile__term" data-translate="xdsl_details_type_label"></div>
+                                    <strong class="oui-tile__description text-uppercase"
+                                            data-ng-bind="access.xdsl.accessType">
+                                    </strong>
                                 </div>
-                                <div>
-                                    <span data-ng-bind="access.xdsl.address.zipCode"></span>
-                                    <span data-ng-bind="access.xdsl.address.city"></span>
+                            </li>
+
+                            <!-- DEGROUPAGE -->
+                            <li class="oui-tile__item">
+                                <div class="oui-tile__definition">
+                                    <div class="oui-tile__term" data-translate="xdsl_details_degroupage_label"></div>
+                                    <div class="oui-tile__description"
+                                         data-ng-if="XdslAccess.lineDetails.deconsolidation !== 'partial' || access.tasks.current.accessPartToTotal"
+                                         data-translate="xdsl_access_deconsolidation_{{XdslAccess.lineDetails.deconsolidation}}"
+                                         data-uib-tooltip-template="'pack-xdsl-access-tooltip-deconsolidation.html'"
+                                         data-tooltip-enable="XdslAccess.lineDetails.deconsolidation"
+                                         data-tooltip-placement="right">
+                                    </div>
+                                    <div class="oui-tile__description"
+                                         data-ng-if="XdslAccess.lineDetails.deconsolidation === 'partial' && !access.tasks.current.accessPartToTotal">
+                                        <a class="service-button"
+                                           data-ui-sref="telecom.pack.xdsl.access-deconsolidation"
+                                           data-uib-tooltip-template="'pack-xdsl-access-tooltip-deconsolidation.html'"
+                                           data-tooltip-enable="XdslAccess.lineDetails.deconsolidation"
+                                           data-tooltip-placement="right">
+                                            <div
+                                                class="service-button-description description-title-small description-title-fullHeight"
```

Comment:
Can be removed `service-button-description description-title-small description-title-fullHeight`?